lib/omni_auth/strategies/miro.rb

```diff
@@ -46,26 +46,6 @@ def token_params
           params[:redirect_uri] = callback_url
         end
       end
-
-      def callback_phase
```

Comment:
Do you know why this was overwritten in the first place?

Reply:
yeah @calvinc03-rewind overwrote it to skip checking for refresh_tokens so we could test miro-onboarding

Reply:
Oh gotcha, makes sense!